release-notes/major12/12.0.0/valtimo-backend-libraries.md

```diff
@@ -4,10 +4,12 @@
 
 The following features were added:
 
-* **New feature1**
+* **valtimo-dependency-versions module**
+
+  A new `valtimo-dependency-versions` module has been created, 
```

Comment:
Although it's kind of obvious what the role is of this module, can you still write that description down here, and provide an possibly obvious example here of why you would want to use this?

Reply:
done